docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Mint NFTs on Avalanche 🚧
+description: Learn how to start minting NFTs on Avalanche.
+issueUrl: https://github.com/protocol/nft-website/issues/224
+---
+
+# Mint NFTs on Avalanche
+
+This short tutorial will quickly guide you through getting started with an EVM-compatible NFT minting work flow on [Avalanche](https://www.avax.network/)using Node.js REPL. This is quite useful if you are migrating from Ethereum or other EVM-compatible blockchain and wish to reuse your code.
```

Comment:
Might be worth some dot points on the agenda for this tutorial here (good for the reader and SEO)

- Setting up avalanche testnet
- ...